test/users-query.ts

```diff
@@ -0,0 +1,454 @@
+import axios from 'axios'
+import { prisma } from '../src/setup-db'
+import { describe, it } from 'mocha'
+import gql from 'graphql-tag'
+import { print } from 'graphql/language/printer'
+import { expect } from 'chai'
+import jwt from 'jsonwebtoken'
+import { faker } from '@faker-js/faker'
+
+describe('Multiple users query mutation tests', function () {
+  const userList = {
+    data: [
+      {
+        name: 'Adler Alves',
+        email: 'Tommie.Medhurst@gmail.com',
+        birthDate: '16-11-1962',
+        password:
+          '$2b$10$5ss0lvGtJIAzKT2ipK35ueAiYBN1Kccd3YJSVgx2WKhWkDM4Udm3e',
+      },
+      {
+        name: 'Bernardete Barros',
+        email: 'Princess.Kiehn52@gmail.com',
+        birthDate: '04-11-1959',
+        password:
+          '$2b$10$DOCXYkeO0QJoOV6mVl94F.CSPla8gMSDXLIIDVKcmptP5MvMslYMG',
+      },
+      {
+        name: 'Claudia Leite',
+        email: 'Daren.Roob0@yahoo.com',
+        birthDate: '01-12-1997',
+        password:
+          '$2b$10$aCBOjaCD2aU8GXIs9Ivia.NWz3iKdXnwO0kXEUCNhQwuyVq9ViX/2',
+      },
+      {
+        name: 'Daniel Almeida',
+        email: 'Jeremie.Bernhard@hotmail.com',
+        birthDate: '28-08-1976',
+        password:
+          '$2b$10$jWb2pfXapMReEI6cqYqVqORg0R9s7H7TNS7iC.W.mz47kt7nbV3Xi',
+      },
+      {
+        name: 'Erick Sousa',
+        email: 'Shyann.Davis20@yahoo.com',
+        birthDate: '12-03-1961',
+        password:
+          '$2b$10$li3MOizIuHQ0gEDF3glM..M2pScu0u2AUntgrynpyg39IR1a.ELHO',
+      },
+      {
+        name: 'Felipe Kalume',
+        email: 'Sedrick.Halvorson19@yahoo.com',
+        birthDate: '22-06-1991',
+        password:
+          '$2b$10$mFRG4CdITKnnij3u/ldlseAC644ENbrdUj.QwKLXO5UTPKkusjMUm',
+      },
+      {
+        name: 'Gabriel Engler',
+        email: 'Obie_Terry85@hotmail.com',
+        birthDate: '04-07-1967',
+        password:
+          '$2b$10$4MUrEPoD.lblZjDWTn8V7.cRZaRyQD6G1HzGqo1Zq74dsJeYp3ofW',
+      },
+      {
+        name: 'Heitor Mascarenhas',
+        email: 'Wyman_Auer@yahoo.com',
+        birthDate: '09-01-1990',
+        password:
+          '$2b$10$kbg91w4/b0PIumY8xL/tpORHmCQFpfUm7i./Rsb7slUmavDTl.SEa',
+      },
+      {
+        name: 'Italo John',
+        email: 'Kelsi_Lowe@hotmail.com',
+        birthDate: '14-01-1995',
+        password:
+          '$2b$10$o7P4MzxFIsvwFAddGMxoyuNGOib1w2xCmVEd.Qu7ggxmU0cpGtXd.',
+      },
+      {
+        name: 'Joao Almeida',
+        email: 'Alvera.MacGyver1@gmail.com',
+        birthDate: '15-04-2003',
+        password:
+          '$2b$10$.xwPoedPtimBYIadcMB0jeH4Hv8lBRWxq5ldWEluxcxwytP1BohJ2',
+      },
+    ],
+  }
+
+  const shuffledList = {
+    data: faker.helpers.shuffle(userList.data),
+  }
```

Comment:
eu recomendaria tirar esse shuffled e criar na ordem que tava ali no `userList` mesmo.
no geral a ideia é evitar tanta aleatoriedade nos testes. Graças a essa abordagem, vc precisou adicionar mais lógica no seu it e meio que replicar o código da paginação  (linha 130, por exemplo)
se vc criasse na ordem que ta, ia poder montar o expectedResponse na mão e iria ter muito mais assertividade no teste, por exemplo para a paginação c ordenação alfabética vc poderia fazer `const expected = [userList[0], userList[1], userList[2]]` 

Dai se for o caso, da uma "embaralhada" manualmente antes de atribuir os valores no array mesmo, por exemplo esse Adler ser o segundo usuario ao invés do primeiro